src/main/scala/codacy/detekt/Detekt.scala

```diff
@@ -101,25 +102,23 @@ object Detekt extends Tool {
           (category.value.toLowerCase, (Map(("active", patternsRaw.nonEmpty)) ++ patterns).asJava)
       }
 
-    val ourConf = Map(("autoCorrect", false), ("failFast", false)) ++ configMapGen
```

Comment:
We lost these "confs", was it an oversight, or on purpose?

Reply:
Passed these configs to the `ProcessingSpec` being created on `ProcessingSettingsFactory`.